cli/popper/commands/cmd_reset.py

```diff
@@ -17,6 +17,7 @@ def cli(ctx):
     pipelines and folders, leaving behind a newly created .popper.yml file.
     """
 
+    click.confirm("Do you want to continue?", abort=True)
```

Comment:
Can we add a sentence like "This will remove all the pipeline files in this project, do you want to continue?"